pkg/services/ssosettings/ssosettingsimpl/service_test.go

```diff
@@ -1440,9 +1440,9 @@ func Test_ProviderService(t *testing.T) {
 			strategiesLength: 2,
 		},
 		{
-			name:                  "should return all fallback strategies and it should return all OAuth providers and saml because the licensing feature is enabled and the provider is setup",
-			isLicenseEnabled:      true,
-			configurableProviders: map[string]bool{"saml": true},
+			name:             "should return all fallback strategies and it should return all OAuth providers and saml because the licensing feature is enabled and the provider is setup",
```

Comment:
nit:
```suggestion
			name:             "should return all fallback strategies and it should return all OAuth providers and SAML because the licensing feature is enabled and the provider is setup",
```